src/DeVinci_frontend/store.ts

```diff
@@ -669,3 +674,11 @@ declare global {
     };
   }
 }
+
+// Add this function at the end of the file
+export function initializeStores() {

```

Comment:
we can initiate downloadedModels like the other variables as well, e.g.
export let userSettings = writable(localStorage.getItem("userSettings"));

so 
export let downloadedModels = writable(JSON.parse(localStorage.getItem("downloadedAiModels")));

the function initializeStores is not needed